template/docker/Dockerfile.j2

```diff
@@ -66,23 +66,55 @@ LABEL io.openshift.tags="ubi9,stackable,sdp,{[ operator.product_string }]"
 LABEL io.k8s.description="Deploy and manage {[ operator.pretty_string }] clusters."
 LABEL io.k8s.display-name="Stackable Operator for {[ operator.pretty_string }]"
 
+COPY <<EOF /etc/dnf/dnf.conf
+[main]
+install_weak_deps=0
+assumeyes=True
+tsflags=nodocs
+EOF
+
 RUN <<EOF
-# Update image and install kerberos client libraries
-# install_weak_deps in microdnf does not support the literal "False" as dnf does
-# https://github.com/rpm-software-management/microdnf/blob/a600c62f29262d71a6259b70dc220df65a2ab9b5/dnf/dnf-main.c#L176-L189
+# Update image and install kerberos client libraries as well as some other utilities
 microdnf update
+
+# **iputils**
+# To make debugging easier, includes things like ping
+# Added 2024-03: We cannot find any vulnerabilities in the past years
```

Comment:
Was this really already added in 2024-03? :)

Reply:
It was over in the docker-images repo, yeah

Reply:
Hmm :thinking: Same with 24.3.0
```
docker run -it --entrypoint bash docker.stackable.tech/stackable/trino-operator:24.7.0                                                                     kind-kind
[stackable@7374d513b091 /]$ ping
bash: ping: command not found
[stackable@7374d513b091 /]$ less
bash: less: command not found
```

Reply:
But honestly I don't care ;)

Reply:
ah! Sorry. It was added in March 2024, not necessarily for the 24.3 release

It should be in 24.7 though

Reply:
It's in neither but anyway ;)

Reply:
Uhm...that's weird. I'll take a look at a later time. Thanks.

Reply:
Just for the record `docker run -it --entrypoint ping docker.stackable.tech/stackable/trino-operator:24.3.0`

Reply:
Ahhh! It has _not_ been added to operators before, only products up until now.